proto/nodeinstance/info/v1/info.proto

```diff
@@ -83,4 +85,17 @@ message NodeInfo {
   reserved 20;
 
   map<string, string> host_labels = 21;
+
+  MachineLearningInfo MLInfo = 22;
+}
+
+message MachineLearningInfo {
+  // true when UpdateNodeInfo.child == true && parent runs ML for child
+  bool ml_on_parent = 1;
```

Comment:
I'm not sure I understand the purpose of this field. This is not something we can interpret or store from the cloud side as is.

Reply:
To elaborate a bit, we need fields that are agnostic to child-parent relationships.

Reply:
`UpdateNodeInstance.MLInfo.ml_on_parent` will be true when the parent provides ML information for the child.

Reply:
> To elaborate a bit, we need fields that are agnostic to child-parent relationships.

Ok, to change this to `ml_accessible`?

Reply:
I'd say at this moment the `MLInfo` structure needs to contain two fields: `ml_capable` and `ml_enabled`.
The `MLInfo`  inside the `NodeInfo` struct should be filled with the info from the node that is collecting the data (i.e. the child node), while the `MLInfo` at the root level should be filled with data from the node that is communicating with the cloud. 
That being said, for routing purposes, the cloud will only use the data stored at the root level, as the claimed node is the only one we have access to.

Reply:
btw `ml_enabled` or `ml_accessible` is ok for me, but just noting that with two fields would be enough, one that indicates the agent can generate ml data (`ml_capable`) and other to say the agent is actually generating it (`ml_accessible` or `ml_enabled`)